app/(setting)/delete/page.tsx

```
@@ -0,0 +1,59 @@
+'use client';
+
+import { useState } from 'react';
+import Image from 'next/image';
+import { useRouter } from 'next/navigation';
+import { FunnelHeadline } from '@/app/(funnel)/components';
+import { FixedButton } from '@/components/ui';
+import { createClient } from '@/lib/supabase/client';
```

ESLint

[error] 'createClient' is defined but never used.

(@typescript-eslint/no-unused-vars)